layouts/partials/widgets/experiences.html

```diff
@@ -0,0 +1,20 @@
+{{ $experiences := $.Param "experiences"}}
+{{ range $experiences }}
+    <div class="mb-6">
+        <div class="bg-secondary-bg rounded border hover:shadow-lg transition ease-in-out duration-200 px-6 pt-6 pb-4">
+            <div class="flex flex-col-reverse lg:flex-row justify-between">
+                <div class="w-full lg:w-2/3">
+                    <div class="mb-4">
+                        <div class="font-bold text-xl">{{ .title }}</div>
+                        <div> {{ .organization | markdownify }}</div>
+                        <span class="mr-8"> {{ .dates | markdownify }}</span> <span> {{ .location | markdownify }} </span>
```

Comment:
I think maybe we can narrow the gap between dates and location? Such as 4.

Reply:
> How about adding a link to organziation?
I think that would be a good idea. I can add an organization url to the front matter and hyperlink it appropriately.

> I think maybe we can narrow the gap between dates and location? Such as 4.
Yes, I think that would look good. I'll do 4 for the gap.

> And do you think it is a good idea to place organization, date, location in one line? Organization at left and the others at right.

Hmm I'm not sure. I would have to try it out.